lib/Dialect/AIEX/IR/AIEXDialect.cpp

```diff
@@ -641,4 +641,24 @@ LogicalResult AIEX::DMAStartBdChainOp::verify() {
     }
   }
   return success();
-}
\ No newline at end of file
+}
+
+//===----------------------------------------------------------------------===//
+// NpuControlPacketOp
+//===----------------------------------------------------------------------===//
+
+uint32_t AIEX::NpuControlPacketOp::getRow() {
+  const auto &targetModel = AIE::getTargetModel(*this);
+  uint32_t addr = getAddress();
+  uint32_t rowInt =
+      (addr & (0xff << targetModel.getRowShift())) >> targetModel.getRowShift();
+  return rowInt;
+}
```

Comment:
This is called `getRow`, and there's a `getColumn` below, but they aren't actually getting row and column, they are doing some adddress calculations. And they are unused in this PR? Can we remove from this PR and add them when there's a use?

Reply:
Sure methods removed from this PR and will be added to a separate PR that follows.